docs/python-reference/index.md

```diff
@@ -0,0 +1,53 @@
+These are autogenerated Python references for the PrairieLearn library. These helpers are available in the Python environment when running a PrairieLearn question.
+
+You can use
+
+```python
+from prairielearn.<submodule> import ...
+```
+
+to use any of these helpers. Any code from `prairielearn.internal` should be considered internal and may change without notice.
+
+!!! warning
+
+    The following imports are now deprecated.
+
+    === "Old"
+
+        ```python
+        import colors
+        import to_precision
+        import python_helper_sympy
+        ```
+
+    === "New"
+
+        ```python
+        import prairielearn.colors
+        import prairielearn.to_precision
+        import prairielearn.sympy_utils
+        ```
+
+## Element Development
+
+These are the core classes for developing elements in PrairieLearn.
```

Comment:
Shouldn't we say these are classes and functions? The classes are really just type hints. I actually don't think they should be included here, as there's a table about each entry of the data dictionary and when they can be modified elsewhere in the documentation.

Reply:
This is fixed in the latest commit.